src/index.ts

```diff
@@ -13,7 +13,7 @@ export {
   mobileNavbarHeight,
 } from './constants';
 export { default as GlobalStyles } from './components/styles/GlobalStyles';
-export { ThemeProvider } from './components/styles/Theme';
+export { ThemeProvider, zopaTheme } from './components/styles/Theme';
```

Comment:
We now export the `zopaTheme` this lets someone extend it easily and allows a more consistent return type in client applications.

Reply:
amazing, we might need that in rezza